### DISCUSSION TITLE
Carapace vs "usage"

### DISCUSSION THREAD

**Discussion by ccoVeille** [carapace-sh/carapace-bin#2520] [General]
I found the following project a few days ago

https://github.com/jdx/usage

It looks like yours. It's slightly different, but you might find in this format something interesting, either as a target format
or something else

https://usage.jdx.dev/spec/

**Comment by ccoVeille** [comment]
I see the question was closed

Could you share your thought about usage project?

**Reply by rsteube** [reply]
Ah, sorry I was just doing some cleaning up :sweat_smile: .
I do have my eyes on it, let's see if it gets some traction.

Converting / bridging the kdl specs should be possible.

**Reply by ccoVeille** [reply]
Great, thanks

Could you reopent he discussion ?

It is currently closed

**Reply by ccoVeille** [reply]
Thanks
